com.microsoft.mrtk.input/Subsystems/Hands/OpenXRHandsSubsystem.cs

```diff
@@ -102,7 +102,10 @@ public override bool TryGetJoint(TrackedHandJoint joint, out HandJointPose pose)
                             return false;
                         }
 
-                        thisQueryValid |= TryUpdateJoint(index, HandJointLocations[HandJointIndexFromTrackedHandJointIndex[index]], playspaceTransform);
+                        var jointLocation = HandJointLocations[HandJointIndexFromTrackedHandJointIndex[index]];
+                        UpdateJoint(index, jointLocation, playspaceTransform);
```

Comment:
What would happen if `UpdateJoint` fails?

Reply:
Looks like it always returns true, maybe we can change it to void.

Reply:
+1 on this

Reply:
Fixed.